js/demo.js

```diff
@@ -34,16 +34,16 @@ $(document).ready(function (e) {
     //
     // Sign-up for free and get your own key: https://graphhopper.com/#directions-api
     //
-    var defaultKey = "bd5f8b44-bfa8-407a-b868-7f2efc1146d9";
```

Comment:
This should be the same as in the[ demo.js of this repo](https://github.com/graphhopper/directions-api-js-client/blob/master/js/demo.js#L37).